### ISSUE TITLE
How to fix translation typo

### ISSUE THREAD

**Issue by nataraj-hates-MS-for-stealing-github** [openstreetmap/merkaartor#285]
Hi!

I'd like to fix translation typo, but  `/merkaartor/translations/README.md` says to use Transifex.com to update translations...
But there is no guide how to do that.

Transifex.com looks like some commercial site, not as anything related to free software...
 
I guess I need some help with fixing that typo.

And it is better to add some guide exlaining how to use Transifex.com in this case in `/merkaartor/translations/README.md`. I can write that short guide, but I need to know how to do it first...

**Comment by Krakonos** [comment]
A short manual:

1. Login with an account/register (easily doable with github, you can use whatever throwaway account you'd like).
2.  When asked about trial/join org, join org. Search for merkaartor.
3. Select your language and proceed.
4. On the dashboard, you have a big blue button to "Translate", use that to start working.

I'll appreciate documentation, though it tends to get out of date rather fast.

I'd also appreciate a free and open source service to run through the same process, but PRs don't work well for translations (people doing translations usually don't appreciate the hassle). I didn't do any recent research, back in the day this was the friendliest platform I could find and already somewhat used in the project when I took it over.

**Comment by stephan75** [comment]
> I'd also appreciate a free and open source service to run through the same process, but PRs don't work well for translations (people doing translations usually don't appreciate the hassle). I didn't do any recent research, back in the day this was the friendliest platform I could find and already somewhat used in the project when I took it over.

@Krakonos ... do I understand right, that you are not happy about using transifex for merkaartor's translations?

If yes: what about giving [weblate.org](https://weblate.org) a try?

It is usable on an own server, or use their server for free if you have a really **open source** project.

See https://wiki.openstreetmap.org/wiki/Translation to find out some OSM related projects already using weblate.

**Comment by Krakonos** [comment]
@stephan75 Correct. It's OK, but recently has been steering into a more closed system than it used to be. I checked out weblate, it appears to be opensource and plenty of projects use it. It also supports .ts files, so I'll give it a try.
